code/obj/item/gun/kinetic.dm

```diff
@@ -952,6 +952,31 @@ ABSTRACT_TYPE(/obj/item/gun/kinetic)
 		..()
 		ammo.amount_left = 0
 		update_icon()
+/obj/item/gun/kinetic/smg
+	name = "Bellatrix submachine gun"
+	desc = "A semi-automatic, 9mm submachine gun, developed by Almagest Weapons Fabrication."
+	icon = 'icons/obj/48x32.dmi'
+	icon_state = "mp52"
+	w_class = 2
```

Comment:
this will fit into normal belts and boxes, is that desired?

Reply:
Hm, copy pasted from the pistol I bet. It should probably be a bit bigger.